src/shortestpaths/astar.jl

```diff
@@ -3,49 +3,60 @@
 
 # A* shortest-path algorithm
 
-function a_star_impl!(g::AbstractGraph,# the graph
-    t, # the end vertex
-    frontier,               # an initialized heap containing the active vertices
-    colormap::Vector{UInt8},  # an (initialized) color-map to indicate status of vertices
-    distmx::AbstractMatrix,
-    heuristic::Function)
+function reconstruct_path!(total_path, # a vector to be filled with the shortest path
+    came_from, # a vector holding the parent of each node in the A* exploration
+    end_idx, # the end vertex
+    g) # the graph
 
     E = Edge{eltype(g)}
```

Comment:
I know you didn't change this line but it really should be `E = edgetype(g)`.

Reply:
Thanks! Changed.